.travis.yml

```diff
@@ -63,15 +68,19 @@ before_install:
 
   - girder_path=$HOME/girder
   - rm -fr $girder_path
-  - git clone https://github.com/girder/girder.git $girder_path && git -C $girder_path checkout a381b5822b1fb063e3d8877fc1a6ff629e52eebf
+  - git clone https://github.com/girder/girder.git $girder_path 
+  # Comment out to use master
+  - git -C $girder_path checkout 75333f56ea17d9bc71747fc84433d52ef394b0f2
   - ln -sf $main_path $girder_path/plugins/
   - ls -l $girder_path/plugins
 
   - CACHE=$HOME/.cache CMAKE_VERSION=3.1.0 CMAKE_SHORT_VERSION=3.1 source $girder_path/scripts/install_cmake.sh
   - cmake --version
 
   - girder_worker_path=$HOME/girder_worker
-  - git clone https://github.com/girder/girder_worker.git $girder_worker_path && git -C $girder_worker_path checkout 4429c6e98b2b11d47edb43390571551ab2d36749
+  - git clone https://github.com/girder/girder_worker.git $girder_worker_path 
+  # Comment out to use master
+  - git -C $girder_worker_path checkout 29c1f504e0f5335d5e6d370bb1077e4df4e05a78
```

Comment:
Also, you've earned the distinction of being the recipient of my very first GH code review! Huzzah! :+1: